docs/src/AerosolActivation.md

```diff
@@ -285,3 +285,100 @@ The CloudMicrophysics package offers an advanced feature for predicting the aero
+```
+
+Here:
+
+- ``N_l``, ``r_l``, and ``G_l`` are the number concentration, radius, and condensational growth coefficient of liquid droplets,
+- ``N_i``, ``r_i``, and ``G_i`` are the corresponding values for ice crystals,
+- ``\rho_w`` and ``\rho_i`` are the densities of liquid water and ice.
+
+We define:
+```math
+K_l = 4 \pi \rho_w N_l r_l G_l, \quad K_i = 4 \pi \rho_i N_i r_i G_i,
+```
+
+yielding the modified supersaturation equation at peak supersaturation:
+```math
+0 = \alpha w - \gamma f(S_{max}) - K_l S_{max} - K_i \big[ \xi (S_{max} + 1) - 1 \big].
+```
+
+### Approximate Solution
+
+Let ``S_0`` be the ARG solution obtained by solving:
+```math
+\alpha w = \gamma f(S_0),
+```
+and approximate the nonlinear sink function ``f(S)`` as linear between 0 and ``S_0``:
+```math
+f(S) \approx f(S_0) \cdot \frac{S}{S_0}.
+```
+
+Substituting into the modified budget and solving for ``S_1``:
+```math
+\alpha w = \gamma f(S_0) \cdot \frac{S_1}{S_0} + K_l S_1 + K_i \big[\xi (S_1 + 1) - 1 \big],
+```
+yields:
+```math
+S_1 = \frac{(\alpha w - K_i (\xi - 1)) S_0}{\alpha w + (K_l + K_i \xi) S_0}.
+```
+
+The final expression for the modified supersaturation becomes:
+```math
+S_{max}^{modified} = \frac{(\alpha w - K_i (\xi - 1)) S_{max}^{ARG}}{\alpha w + (K_l + K_i \xi) S_{max}^{ARG}}.
+```
+
+- When ``K_l = K_i = 0``, the formula reduces to the original ARG solution.
+- Larger values of $K_l$ or $K_i$ reduce the supersaturation due to vapor uptake by preexisting hydrometeors.
+- This extension allows the ARG activation scheme to be applied at any vertical level, making it suitable for use as a local operator in models that resolve cloud-scale dynamics.
+
+### Example
+
+To illustrate the behavior of the modified ARG scheme, we compare the maximum supersaturation it predicts against that from a detailed parcel model in four idealized scenarios:
+
+- Varying the liquid droplet number concentration with no ice particles present,
+- Varying the ice crystal number concentration with no liquid droplets present,
+- Varying the initial droplet radius with no ice particles present,
+- Varying the initial ice particle radius with no liquid droplets present.
+
+In all cases, the parcel model assumes an initial zero supersaturation and includes activation, condensational growth, and depositional growth during adiabatic ascent. A background population of sulfate aerosol with an initial concentration of 500 cm⁻³ is present in all scenarios. For simplicity, ice particles are assumed to be monodisperse. Liquid droplets formed through activation and those present initially are represented as two separate monodisperse populations.
+
+The figure below shows the ratio of the maximum supersaturation reached in the parcel model to that predicted by the modified ARG scheme. This comparison highlights the conditions under which the modified ARG provides a good approximation to the more detailed parcel-model results.
+
```

Comment:
Maybe it would be good o leave a sentence here about the need for additional clipping when coupled to a full model and treating it as a local parameterization?